pyanaconda/storage/execution.py

```diff
@@ -120,15 +123,23 @@ def clear_partitions(storage):
     storage.clear_partitions()
 
 
-class AutomaticPartitioningExecutor(object):
-    """The executor of the automatic partitioning."""
+class PartitioningExecutor(object):
```

Comment:
Ideally you should also inherit from ABC to avoid base class instantiation.

Reply:
Fixed.